src/settings/handlers/folder/CellSizeDropDownHandler.ts

```diff
@@ -0,0 +1,30 @@
+import { CellSizeOptions } from "helpers/Constants";
+import { AbstractSettingsHandler, SettingHandlerResponse } from "settings/handlers/AbstractSettingHandler";
+import { add_dropdown } from "settings/SettingsComponents";
+
+export class CellSizeDropDownHandler extends AbstractSettingsHandler {
+    settingTitle = 'Cell size';
+    handle(settingHandlerResponse: SettingHandlerResponse): SettingHandlerResponse {
+        const { settingsManager, containerEl, view } = settingHandlerResponse;
+        const source_dropdown_promise = async (value: string): Promise<void> => {
+            // update settings
+            view.diskConfig.updateConfig('cell_size', value);
+            // Force refresh of settings
+            settingsManager.reset(settingHandlerResponse);
```

Comment:
Do you need to refresh the settings component when you change the option?
Its used if there is some c¡kind of dependency between another config (disable/enable or stuff like that)

Reply:
Oh, no, I just copied this code from the other settings handlers. According to your explanation, I believe it can be removed.

Reply:
oky. Just remove the line 13 and PR is in =)

Reply:
Done!